sns/README.md

```diff
@@ -15,23 +15,288 @@ Current version is `0.1.0`.
 
 This object MUST NOT contain any properties. Its name is reserved for future use.
 
-
-
 <a name="channel"></a>
 
 ## Channel Binding Object
 
-This object MUST NOT contain any properties. Its name is reserved for future use.
+This object contains information about the channel representation in SNS.
+
+We represent an AsyncAPI Channel with a Topic in SNS. The bindings here allow definition of a topic within SNS. We provide properties on the binding that allow creation of a topic in infrastructure-as-code scenarios. Be aware that although the binding offers that flexibility, it may be more maintainable to specify properties such as SNS Access Control Policy outside of AsyncAPI.
+
+SNS supports many optional properties. To mark a channel as SNS, but use default values for the channel properties, just use an empty object {}. 
+
+### Fields
+
+|Field Name | Type | Description|
+|---|:---:|---|
+| <a name="channelBindingObjectName"></a>`name` | string | **Required.** The name of the topic. Can be different from the channel name to allow flexibility around AWS resource naming limitations.|
+| <a name="channelBindingObjectFifo"></a>`type` | [ordering](#ordering)| **Optional.** By default, we assume an unordered SNS topic. This field allows configuration of a FIFO SNS Topic.  |
```

Comment:
Perhaps we should rename this property to `ordering` instead of `type` as type is a reserved keyword in many languages that might want to make use of this schema and it's also a reserved word in JSON schema and can easily become confusing. It's quite overloaded.